lib/db/redis.js

```diff
@@ -73,6 +75,29 @@ RedisDB.prototype.tail = function () {
   return P.resolve('0')
 }
 
+RedisDB.prototype.set = function (key, value, ttl) {
```

Comment:
Alternately, maybe this `'_s_'` prefixing should be lifted into the sbuscriptions module so that this is more formally a generic k/v API.